db/schema.rb

```diff
@@ -37,8 +40,8 @@
     t.string "email"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "course_id"
-    t.integer "user_id"
+    t.bigint "course_id"
+    t.bigint "user_id"
```

Comment:
again wondering what issue this fixed... is there some library assuming that these are bigints or are we worried about having more than 2^32 courses? :)

Reply:
This was changed accidentally, by something I was playing around with. Im not sure what exactly caused this to happen but ill change it back